ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -295,14 +295,19 @@ export const WorkspaceCard = fp.flow(withNavigation)(
                     Last Changed: {displayDate(workspace.lastModifiedTime)}
                   </div>
                 </FlexColumn>
-                <FlexColumn style={{justifyContent: 'flex-end', alignItems: 'flex-end', direction: 'rtl', marginLeft: 15}}>
-                  {accessTierShortName === AccessTierShortNames.Controlled && <ControlledTierBadge/>}
+                <FlexColumn style={{justifyContent: 'flex-end', marginLeft: '0.8rem'}}>
```

Comment:
since the only element inside this FlexColumn is a FlexRow, can we combine them?

Reply:
Combining them makes the lock and CT badge parallel to the OWNER tag rather than the same row as that of LastModified
<img width="335" alt="Screen Shot 2021-11-18 at 11 01 22 AM" src="https://user-images.githubusercontent.com/34481816/142451094-b8ed5e20-2b94-4be2-9a85-fb6be97d6264.png">

Reply:
hm ok